packages/core/src/components/discord-reaction/discord-reaction.css

```diff
@@ -0,0 +1,71 @@
+.discord-reaction {
+	border-radius: 0.5rem;
+	cursor: pointer;
+	flex-shrink: 0;
+	margin-right: 0.25rem;
+	margin-bottom: 0.25rem;
+	user-select: none;
+	transition: none 0.1s ease;
+	transition-property: background-color, border-color;
+	background-color: #2f3136;
+	border: 1px solid transparent;
+}
+
+.discord-light-theme .discord-reaction {
+	background-color: #f2f3f5;
+}
+
+.discord-reaction:hover {
+	background-color: #36393f;
+	border-color: #fff2;
+}
+
+.discord-light-theme .discord-reaction:not(.discord-reaction-reacted):hover {
+	background-color: white;
+	border-color: #0003;
+}
+
+.discord-reaction.discord-reaction-reacted {
+	background-color: #5865f226;
```

Comment:
This is either an invalid colour hex, or a colour hex with alpha value. Assuming the latter, please change it to its respective `rgba` because colour hex with alpha is really confusing to read.